modules/ppcp-onboarding/assets/js/onboarding.js

```diff
@@ -88,7 +88,7 @@ const ppcp_onboarding = {
                 return res.json();
             }).then((data)=>{
                 if (!data.success) {
-                    console.error(data);
+                    alert('Could not update signup buttons: ' + data);
```

Comment:
Displaying objects like this will not work, it will be `[object Object]`.

```
const data = {a: [1, 2]}
alert('error: ' + data)
```

You can use `JSON.stringify(data)`